packages/vue/src/components/authenticator-sign-up-form-fields.vue

```diff
@@ -57,11 +57,14 @@ const loginMechanism = fieldNames.shift() as LoginMechanism;
 const actorContext = computed(() => {
   return getActorContext(state.value);
 }) as ComputedRef<SignInContext>;
-const formValues = actorContext.value?.formValues as AuthFormData;
+const formValues = computed(
+  () => actorContext.value?.formValues as AuthFormData
+);
+actorContext.value.formValues;
```

Comment:
Why is `actorContext.value.formValues` accessed here, since it doesn't look like it does anything?

Is it here for the Proxy to get accessed and trigger an initial computation?

Reply:
oops, I'll delete that. Not sure how that got in there.